server/models/blogs.js

```diff
@@ -0,0 +1,34 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const BlogsSchema = new Schema({
```

Comment:
Timestamps?

Reply:
Right! That would be useful here. Including it